packages/components/src/card/styles.js

```diff
@@ -27,7 +24,7 @@ export const Header = css`
 export const Footer = css`
 	border-top: 1px solid;
 
-	&:first-child {
+	&:first-of-type {
```

Comment:
This was causing a test failure due to console logging 😒

Reply:
As far as I remember, Emotion prepends `<style>` elements when doing SSR, and that's why `:first-child` may fail. Is that correct?

Reply:
Something like that yes.